.github/workflows/android-app.yml

```
@@ -75,7 +75,7 @@ jobs:
 
   debug:
     name: Build unsigned APKs
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on:            ubuntu-20.04
```
I would suggest to use `ubuntu-20.04` explicitly, as it is expected that `ubuntu-latest` will start pointing to `ubuntu-22.04` soon. https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/